dts/bindings/w1/zephyr,w1-gpio.yaml

```diff
@@ -6,19 +6,25 @@ description: |
 
   This defines a one-wire driver through GPIO bit-banging.
 
+
   For example:
 
   / {
         w1: w1 {
                  compatible = "zephyr,w1-gpio";
-                 gpios = <&gpio0 13 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN | GPIO_PULL_UP)>;
+                 gpios = <&gpio0 13 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
```

Comment:
Thanks for noticing that this has currently no effect.
I have planned to add support for seperate rx/tx pins and there one is expected to set the drive flags via dts.
What do you think about leaving it here for now and I'll send a separate commit removing it from the driver including a notice in the release notes?